src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Allowed metric instrument names to contain `/` characters. ([#4882](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4882))
```

Comment:
nit: Would you move this below the `AggregatorStore` entry? Typically we do these in chronological order.